flink-connectors/flink-connector-kafka-base/pom.xml

```diff
@@ -44,12 +44,6 @@ under the License.
 
 		<!-- core dependencies -->
 
-		<dependency>
```

Comment:
We need to reintroduce this dependency as `org.apache.flink.streaming.util.serialization.JSONKeyValueDeserializationSchema` still requires Jackson. I discussed this with Chesnay. I will reintroduce it while merging.